src/code/z_fbdemo.c

```diff
@@ -40,6 +40,22 @@ Gfx sTransTileSetupDL[] = {
     gsSPEndDisplayList(),
 };
 
+#define SET_VERTEX(vtx, ob0, ob1, ob2, f, tc0, tc1, n0, n1, n2, alpha) \
+    {                                                                  \
+        Vtx_tn* vtxn = &(vtx)->n;                                      \
+        vtxn->ob[0] = (ob0);                                           \
```

Comment:
does this make sense to use in more places?
Believe the answer is yes but it doesnt match in all cases, idr

Reply:
I'm really interested in the answer being yes but I've seen enough subtle differences to have my doubts

I hope, though

Reply:
I think this is the only place it would be used (specifically for `vtx->n`, not `vtx->v`). I'm not sure about the `v` counterpart, there's a lot of scary code in e.g. z_parameter.c which I think I don't want to mess with right now